app/fellow.py

```diff
@@ -0,0 +1,63 @@
+from os import sys, path
+
+from app.person import Person
+
+sys.path.append(path.dirname(path.dirname(
+    path.dirname(path.abspath(__file__)))))
+
+
+class Fellow(Person):
+    """docstring for Fellow"""
+
+    fellow_list = []
+    unallocated_fellows = {"office": [], "livingspace": []}
+
+    def __init__(self, lname, fname, wants_accomodation=False):
+        super(Fellow, self).__init__(lname, fname)
+        self.category = "fellow"
+        self.livingspace = None
+        self.wants_accomodation = wants_accomodation
+
+    def set_assigned_livingspace(self, livingspace):
+        self.livingspace = livingspace
+
+    def get_assigned_livingspace(self):
+        return self.livingspace
+
+    def set_wants_accomodation(self, param):
+        self.wants_accomodation = param
+
+    def get_wants_accomodation(self):
+        return self.wants_accomodation
+
+    def set_category(self, new_category):
+        self.category = new_category.lower()
+
+    def get_category(self):
+        return self.category
```

Comment:
checkout Python getters and setters..

http://www.python-course.eu/python3_properties.php